hw/ip/otbn/doc/dv/index.md

```diff
@@ -173,6 +173,21 @@ See the instruction counter saturate.
 
 > This is tracked in the `insn_cnt_if` interface with the `InsnCntSaturated_C` cover property.
 
+#### External (bus-accessible) CSRs
+
+The OTBN block exposes functionality to a bus host through bus-accessible CSRs.
+Behavior of some CSRs depends on [OTBN's operational state]({{< relref ".#design-details-operational-states" >}}).
+
+Events we want to see:
```

Comment:
To match the instruction-based stuff below, could you name the covergroup for each bullet point? The idea is that we'll have a really tedious afternoon at some point where we read through the English descriptions and match them up with the coverage collection, convincing ourselves that it really does do what we think...